i18n/README.es.md

```diff
@@ -147,8 +147,7 @@ Nuestra librería de cliente es modular. Cada sub-librería es una implementaci
   <tr>
     <td>Kotlin</td>
     <td><a href="https://github.com/supabase-community/supabase-kt" target="_blank" rel="noopener noreferrer">supabase-kt</a></td>
-    <td><a href="https://github.com/supabase-community/postgrest-kt" target="_blank" rel="noopener noreferrer">postgrest-kt</a></td>
-    <td><a href="https://github.com/supabase-community/gotrue-kt" target="_blank" rel="noopener noreferrer">gotrue-kt</a></td>
+    <td><a href="https://github.com/supabase-community/supabase-kt/tree/master/Realtime" target="_blank" rel="noopener noreferrer">realtime-kt</a></td>
```

Comment:
I think this got messed up a little. Realtime is in there twice now, postgrest and gotrue are missing

Reply:
Yep, that happens when trying to update non-matching READMEs in bulk. Should be fixed now